apps/src/lib/kits/maker/boards/FakeMBBoard.js

```diff
@@ -0,0 +1,208 @@
+/** @file Fake for running Maker apps without an attached MicroBit board. */
+import {EventEmitter} from 'events'; // provided by webpack's node-libs-browser
+import {MBFirmataClientStub} from '../../../../../test/unit/lib/kits/maker/boards/makeStubBoard';
```

Comment:
It feels a little odd to have test code in production for a couple reasons.
- It makes it a little harder to separate deploy packages from test packages (although I'd guess WebPack handles this gracefully for us, it's a bit of an odd pattern, so we may run into odd edge cases as we do version bumps in the future)
- When updating makeStubBoard in order to update a test, a maintainer wouldn't expect it to break production code since it's in a /test directory. 

A pattern I've seen work well in the past is to put these types of shared files in the production directory and have the tests import from there instead of the other way around. I've also seen a production util directory used for this as well.

Reply:
This makes sense - thanks for the explanation. I'll go ahead and move `makeStubBoard.js` to src/lib/kits/maker/util.  While I'm here, I'll also rename the `makeStubBoard` function as `CPBoardStub`. Thanks!